<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Sender BSC wallet </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>30</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wallet Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>89</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Private Key:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>139</y>
      <width>140</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wallets To Send:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WalletsButton">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="FilenameLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>271</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>247</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send Amounts:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FixedRadioButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>254</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fixed Amount</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RandomRadioButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>254</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Random Amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>284</y>
      <width>47</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>324</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>194</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Token Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PrivateKeyEditText">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>87</y>
      <width>370</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Logs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FixedLineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>286</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="FromLineEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>285</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="ToLineEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>328</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="WalletLineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>370</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="LogsTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="TokenButton">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>196</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="TokenLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>196</y>
      <width>271</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
